### PR TITLE
added ASH-XIIX keyboard

### PR DESCRIPTION
**Pull request by sh-xguitar**
## Description
Added ASH-XIIX

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ash_xiix/rules.mk [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no      # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes          # USB Nkey Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+MIDI_ENABLE = no           # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no        # Unicode
+BLUETOOTH_ENABLE = no      # Enable Bluetooth
+AUDIO_ENABLE = no          # Audio output
```
Comment (zvecr) [inline keyboards/ash_xiix/rules.mk:24]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/ash_xiix/config.h [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0xFEED
+#define PRODUCT_ID   0x0002
+#define DEVICE_VER   0x0001
+#define MANUFACTURER ASH
+#define PRODUCT      XIIX
+#define DESCRIPTION Cherry G80/G81 1800 replacement PCB
```
Comment (fauxpark) [inline keyboards/ash_xiix/config.h:25]:
```suggestion
```

File: keyboards/ash_xiix/config.h [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0xFEED
+#define PRODUCT_ID   0x0002
```
Comment (fauxpark) [inline keyboards/ash_xiix/config.h:21]:
```suggestion
#define PRODUCT_ID   0x18D2
```
```
echo -n "XIIX" | sha256sum | cut -c1-4
```

**Comment by sh-xguitar** [comment]
I was wondering if someone give me VENDOR_ID

File: keyboards/ash_xiix/config.h [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0xFEED
+#define PRODUCT_ID   0x18D2
```
Comment (fauxpark) [inline keyboards/ash_xiix/config.h:21]: Sorry, I used the wrong command to generate this. It should be:
```suggestion
#define PRODUCT_ID   0x14BC
```
```
echo -n "XIIX" | shasum | cut -c1-4
```

File: keyboards/ash_xiix/info.json [hunk 5]
```diff
@@ -0,0 +1,540 @@
...
+                {
+                    "label": "down",
+                    "x": 14.25,
+                    "y": 6.25
+                },
+                {
+                    "label": "right",
+                    "x": 15.25,
+                    "y": 6.25
+                }
```
Comment (noroadsleft) [inline keyboards/ash_xiix/info.json:536]: And here.

```suggestion
                {
                    "label": "left",
                    "x": 13.25,
                    "y": 6.25
                },
                {
                    "label": "down",
                    "x": 14.25,
                    "y": 6.25
                },
                {
                    "label": "right",
                    "x": 15.25,
                    "y": 6.25
                },
                {
                    "label": "0",
                    "x": 16.5,
                    "y": 6
                },
                {
                    "label": ".",
                    "x": 17.5,
                    "y": 6
                }
```

File: keyboards/ash_xiix/readme.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# ASH-XIIX
+
+In mission to fill my Cherry 1800 case. So I continue developing 1800 PCB (ASH-1800) to fit G80-1800 case. It has some update to make ASH-XIIX more user friendly.
+
+![ASH-XIIX](https://i.imgur.com/KJk9dDo.jpg)
```
Comment (noroadsleft) [inline keyboards/ash_xiix/readme.md:5]:
```suggestion
![ASH-XIIX](https://i.imgur.com/KJk9dDol.jpg)

In mission to fill my Cherry 1800 case. So I continue developing 1800 PCB (ASH-1800) to fit G80-1800 case. It has some update to make ASH-XIIX more user friendly.
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**